activerecord/test/cases/database_configurations/hash_config_test.rb

```diff
@@ -69,6 +69,21 @@ def test_idle_timeout_nil_when_less_than_or_equal_to_zero
         config = HashConfig.new("default_env", "primary", idle_timeout: "0")
         assert_nil config.idle_timeout
       end
+
+      def test_default_schema_dump_value
+        config = HashConfig.new("default_env", "primary", { schema_dump: nil })
+        assert_equal config.schema_dump, true
+      end
+
+      def test_schema_dump_value_set_to_true
+        config = HashConfig.new("default_env", "primary", { schema_dump: true })
+        assert_equal config.schema_dump, true
```

Comment:
These assertions are backwards. It should be:

```suggestion
        assert_equal true, config.schema_dump
```